octopus-server/src/main/resources/buildServerResources/editOctopusMetadata.jsp

```diff
@@ -89,9 +89,13 @@
   </tr>
 
   <tr class="advancedSetting">
-    <th>Force overwrite existing metadata:</th>
+    <th>Overwrite Mode:</th>
     <td>
-      <props:checkboxProperty name="${keys.forcePushKey}" />
+      <props:selectProperty name="${keys.forcePushKey}">
+        <props:option value="false">Fail If Exists</props:option>
+        <props:option value="true">Overwrite Existing</props:option>
```

Comment:
I'm assuming these two values are `true/false` for backwards compatibility?

Reply:
It is, we haven't worked out how to migrate data, so without this we'd break existing steps.